test/fixtures/state-saved-v1-4.js

```diff
@@ -149,6 +150,9 @@ export const stateSavedV1 = {
           brush: {
             size: 0.5,
             enabled: false
+          },
+          geocoder: {
```

Comment:
the saved state should not be changed, the merged config should have geocoder set to `{enablced: false}`